docs/API.md

```diff
@@ -2,6 +2,21 @@
 
 TODO: formally document supported HTTP status codes, format of error and success responses, HTTPS cookie requirements, and which requirements are relaxed when developing locally.
 
+### Auth API
+
+Chronicle uses Firefox Accounts to log users in, but manages its own session duration.
+
+The session cookie is encrypted server-side, so it is not accessible to the client-side code.
+
+The Firefox Accounts OAuth flow is handled by the server; the front-end just needs to redirect logged-out users to the `/auth/login` endpoint.
+
+Note, if the `testUser` config option is set, the cookie will be created directly by visiting `/auth/login`, ignoring the oauth step.
+
+### Auth API Methods
+
+- Start the login flow: `GET /auth/login`
+- Logout: `GET /auth/logout`
+
```

Comment:
I was pretty tired by the time I got around to adding the docs, so they are looking pretty thin. I'm happy to add more docs here, now or later; comments welcome.